docs/guides/integration/docker.md

```diff
@@ -31,9 +31,44 @@ Note this requires `curl` to be available.
 
 In either case, it is best practice to pin to a specific uv version.
 
-## Installing a package
+## Installing a project
+
+If you're using uv to manage your project, you can copy it into the image and install it:
+
+```dockerfile title="Dockerfile"
+# Copy the project into the image
+ADD . /app
+WORKDIR /app
```

Comment:
Maybe overkill, but if setting `WORKDIR` before `ADD`, we can copy in `./` without repeating `/app`:
```suggestion
# Set the current working directory to `/app`
WORKDIR /app

# Copy the project into the image current working directory
ADD . ./
```

Reply:
I think I'd write it the way I have now (personally) maybe it's a little clearer?

Reply:
Yep maybe better to be explicit here indeed.